tools/db_stress.cc

```diff
@@ -2360,8 +2360,8 @@ int main(int argc, char** argv) {
   SetUsageMessage(std::string("\nUSAGE:\n") + std::string(argv[0]) +
                   " [OPTIONS]...");
   ParseCommandLineFlags(&argc, &argv, true);
-#if !defined(OS_MACOSX) && !defined(OS_WIN) && !defined(OS_SOLARIS) &&  \
```

Comment:
as commented on #2337, I don't understand why we mask O_DIRECT on platforms that don't support it, rather than just not running with direct reads/writes, which would be a lot clearer to me.

for this change, once we stop masking in release mode, but still try to enable direct reads/writes on platforms that don't support it, I expect we'll get some runtime error?